manage.py

```diff
@@ -70,12 +74,16 @@ def add_fake_data(number_users):
     Adds fake data to the database.
     """
     User.generate_fake(count=number_users)
-    Resource.generate_fake()
```

Comment:
I think `add_seattle_data` should be disjoint, since it's not fake data. See my suggestion changes to the readme in #9.